bundle-types/action.yml

```diff
@@ -0,0 +1,116 @@
+name: "Grafana Bundle Types"
+description: "Bundles a Grafana plugins types and pushes them to npm."
+
+outputs:
+  typesFilePath:
+    description: "The path to the built types file."
+    value: ${{ steps.set-outputs.outputs.typesFilePath }}
+
+inputs:
+  entryPoint:
+    description: "Location of types file to bundle."
+    required: false
+    default: "./src/types/index.ts"
+  tsConfig:
+    description: "A path to the tsconfig file to use when bundling types."
+    required: false
+    default: ""
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@v4
+      with:
+        node-version: "${{ inputs.node-version }}"
```

Comment:
there's no such input

Reply:
Well spotted! Added in https://github.com/grafana/plugin-actions/pull/40/commits/dbf6b84cfb4f75fccb193fbd48c8b29a7a50d476